src/apps/components/run.js

```diff
@@ -83,21 +83,12 @@ class RunApp extends React.Component {
     render() {
         return (
             <div className="app-main">
-                <div className="main-view">
-                    <JuttleViewer bundle={this.props.bundle} />
+                <RunHeader {...this.props} onRunClick={this.handleRunClick} ref="runHeader" />
+                <div className="run-main">
                     <div ref="juttleSource"></div>
                     <div ref="juttleViewLayout"></div>
                     <ErrorView error={this.props.error} />
                 </div>
-                <div className="right-rail">
-                    <div ref="juttleInputsContainer" onKeyDown={this._onInputContainerKeyDown}></div>
```

Comment:
alright fixed and added test: https://github.com/juttle/juttle-viewer/commit/7cc1d8f9d91254c075cc24b2af70ea51ca51ad25

my experience with these features is if you don't test them, you're bound to lose them at some point.